### DISCUSSION TITLE
A classic Aperol spritz is typically served with a slice of what fruit?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#460] [Galaxy Trivia] [answered]
**Bonus:** What slice of fruit is added to a classic *Moscow Mule*?

**Comment by peterbaccaro** [comment, marked as answer]
A **slice of fresh orange** is the classic garnish for an Aperol spritz.
A **slice of fresh lime** is the classic garnish for a Moscow Mule.
